src/gui/optionsdialog.ui

```diff
@@ -1927,8 +1961,11 @@
                  <layout class="QGridLayout" name="gridLayout_7">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_6">
+                    <property name="toolTip">
+                     <string>Start time</string>
+                    </property>
                     <property name="text">
-                     <string comment="From start time">From:</string>
+                     <string extracomment="From start time">From:</string>
```

Comment:
This should be reverted.

Reply:
Oops, I missed these, pushing after work

Reply:
Done.